inventory/docker-compose.yml

```diff
@@ -0,0 +1,21 @@
+version: '3.1'
+services:
+  web:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    command: python manage.py runserver 0.0.0.0:8000
+    volumes:
+      - .:/inventoryapp
+    ports:
+      - "8000:8000"
+    depends_on:
+      - db
+  db:
+    image: postgres:15
+    environment:
+      POSTGRES_DB: AirQoHardWare
+      POSTGRES_USER: postgres
+      POSTGRES_PASSWORD: P@ss1234
```

Comment:
Same with these, are we supposed to show our passwords in this public repo?
And how come nothing in the repo is flagging this issue? How did this PR manage to bypass Git guardian on his local machine? etc?

 @OlukaGibson / @BenjaminSsempala / @Psalmz777